### PR TITLE
Cssedform

### PR DESCRIPTION
**Pull request by Fbouchezpro**

### CODE REVIEW

**Review by AlexisFLT — APPROVED** [review]
fine by me

File: frontend/src/components/Form/style.css [hunk 1]
```diff
@@ -1,10 +1,11 @@
-.containerForm {
+/* .containerForm {
```
Comment (LoicBrassart) [inline frontend/src/components/Form/style.css:1]: ...

File: frontend/src/components/Formradio/index.jsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+import "./style.css";
+
+export default function Formradio() {
+  const [movies, setMovies] = useState([]);
+  const [selectedRadio, setSelectedRadio] = useState("");
+  const radios = [
+    {
+      name: "Action",
+      id: 1,
+      genre: 28,
```
Comment (LoicBrassart) [inline frontend/src/components/Formradio/index.jsx:13]: Already read this code on the other PR

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
See other PR + Don't duplicate code between branches

**Comment by LoicBrassart** [comment]
Mr @AlexisFLT , please **read** your PR, I don't think you did it properly this time

**Pull request closed without merging**